src/geckomat/model_adapter/ModelAdapter.m

```diff
@@ -1,41 +1,59 @@
 %Abstract Base class for adapters for different species
-classdef (Abstract) ModelAdapter 
-	methods (Abstract)
-		[spont,spontRxnNames] = getSpontaneousReactions(obj,model);
+classdef (Abstract) ModelAdapter
+    methods (Abstract)
+        [spont,spontRxnNames] = getSpontaneousReactions(obj,model);
     end
-	methods
+    methods
         %not really needed, we could access it directly. Not very nice, but practical
         function parameters = getParameters(obj)
             parameters = obj.params;
         end
-        
-        %This allows for changing parameters in an adapter without 
+
+        %This allows for changing parameters in an adapter without
         %creating a new class. May be convenient, although less clean
         %function obj = setParameters(obj, parameters)
         %    obj.mParams = parameters;
         %end
-        
-		function model = manualModifications(obj,model) %default is to do nothing
+
+        function model = manualModifications(obj,model) %default is to do nothing
         end
-        
+
         %The genes returned here should match the gene id set in the parameter params.uniprotGeneIdField
         function genes = getUniprotCompatibleGenes(obj,inGenes)
             genes = inGenes;
         end
-        
+
+        function uniprotIDs = getUniprotIDsFromTable(obj,modelGenes)
+            conversionTable = fullfile(obj.params.path,'data','uniprotConversion.tsv');
+            if exist(conversionTable,'file')
+                fID=fopen(conversionTable,'r');
+                conversionTable = textscan(fID,'%q %q','Delimiter','\t','HeaderLines',1);
```

Comment:
The text in the `uniprotConversion.tsv` file cannot be read.
The resulting `conversionTable` is just 1x2 cell, i.e., `[] []`